www/class/centreonLogAction.class.php

```diff
@@ -39,6 +39,11 @@ class CentreonLogAction
     protected $logUser;
     protected $uselessKey;
 
+    /**
+     * Const use to keep the changelog mechanism with hidden password values
+     */
+    const PASSWORD_BEFORE = '*******';
```

Comment:
can you use CentreonAuth::PWS_OCCULTATION?

Reply:
I'll use it but for PASSWORD_AFTER, because it is needed to PASSWORD_BEFORE as an amount of asterisks different of PASSWORD_AFTER.